src/pages/dan/Main/Feed/CommentList.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import Comment from './Comment';
+import './CommentList.scss';
+
+class CommentList extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentList: [],
+      comment: '',
+    };
+  }
+
+  // Comment state
+  textChange = event => {
+    this.setState(
+      {
+        comment: event.target.value,
+      },
+      () => {
+        this.commentValid();
+      }
+    );
+  };
+
+  // Comment validation
+  commentValid() {
+    const { comment } = this.state;
+    return comment.length > 0 ? true : false;
+  }
+
+  // Comment 추가 후 textarea 리셋
+  addComment = () => {
+    if (this.commentValid() === true) {
+      this.setState({
+        commentList: this.state.commentList.concat({
+          comment: this.state.comment,
+        }),
+        comment: '',
+      });
+    }
+  };
```

Comment:
```suggestion
  addComment = () => {
  const { comment, commentList } = this.state;
    if (this.commentValid() === true) {
      this.setState({
        commentList: commentList.concat({
          comment: comment,
        }),
        comment: '',
      });
    }
  };
```
이렇게 구조분해 할당을 적용해보는건 어떤지 조심스럽게 올려봅니다!

Reply:
늘 놓치는 구조분해 할당! 자세히 알려주셔서 감사합니다! 내용 참고하여 수정진행 하겠습니다!👍🏽